butane_core/src/db/pg.rs

```diff
@@ -583,6 +596,16 @@ fn create_table_constraints(table: &ATable) -> String {
         .join("\n")
 }
 
+fn remove_table_constraints(table: &ATable) -> Result<String> {
```

Comment:
Method name is a bit misleading. It doesn't remove all constraints, it removes only the foreign key constraints. Not unique/primary key/etc.